crates/astria-sequencer-relayer/tests/blackbox/main.rs

```diff
@@ -51,14 +51,11 @@ async fn report_degraded_if_block_fetch_fails() {
         actual: readyz.json::<serde_json::Value>().await.unwrap(),
     );
 
+    // don't mount another block, so the relayer will fail to fetch the block
     let abci_guard = sequencer_relayer.mount_abci_response(1).await;
-    let block_guard = sequencer_relayer.mount_bad_block_response(1).await;
```

Comment:
This made this test more shaky (flaky?) then before, because it relies on relayer attempting to fetch a new bock within `2 * block_time` failing, and reporting that.

I think it's fine for this PR, but could you add a big `NOTE` or even `XXX` to make note of that?